tests/core/test_cost_calculation.py

```diff
@@ -103,16 +106,15 @@ async def test_strict_mode(self):
17ff2f808080ffff01ffff09808080ff01808080ffff01ffff05ffff05ff04ffff05ff05ffff05ffffff05ff06ffff05ff02ffff05ff17ffff01ff808080808080ffff01ff8080808080ffffff05ff17ff2f8080808080ff01808080ffff05ffff01ffff39ffff05ffff04ffff08ff0580ffff01ffff0bffff01ff0280ffffff05ff06ffff05ff02ffff05ff09ffff01ff808080808080ffffff05ff06ffff05ff02ffff05ff0dffff01ff8080808080808080ffff01ffff0bffff01ff0180ff05808080ff01808080ff0180808080ffff05ffff01ffb08f5c4a3a8ee47f20d3ff05720dd00c2c3433153e8f1ef62a8bb247208b9e12fc17a57e3a8cd6dcd58a720efe0f39747280ff01808080ffff80ffff01ff8080ff80808080ffffa0476fb2043f99b8ad62849d66fc3af9b0b9e76930c5a700c29c2cbd733b931466ffffffff05ffff01ffffff05ffff01ffffff05ffff04ff0bffff01ffffff05ffff04ffff0aff05ffff12ff0bffff13ffff0bff0bffffff05ff06ffff05ff02ffff05ff17ffff01ff80808080808080808080ffff01ffffff05ff17ff2f808080ffff01ffff09808080ff01808080ffff01ffff05ffff05ff04ffff05ff05ffff05ffffff05ff06ffff05ff02ffff05ff17ffff01ff808080808080ffff01ff8080808080ffffff05ff17ff2f8080808080ff01808080ffff05ffff01ffff39ffff05ffff04ffff08ff0580ffff01ffff0bffff01ff0280ffffff05ff06ffff05ff02ffff05ff09ffff01ff808080808080ffffff05ff06ffff05ff02ffff05ff0dffff01ff8080808080808080ffff01ffff0bffff01ff0180ff05808080ff01808080ff0180808080ffff05ffff01ffb08f5c4a3a8ee47f20d3ff05720dd00c2c3433153e8f1ef62a8bb247208b9e12fc17a57e3a8cd6dcd58a720efe0f39747280ff01808080ffff80ffff01ff8080ff80808080ffffa0c504e728a7bd669ff56f57d98e1b3628b47b64dd588b8611afe041e8ea951b6effffffff05ffff01ffffff05ffff01ffffff05ffff04ff0bffff01ffffff05ffff04ffff0aff05ffff12ff0bffff13ffff0bff0bffffff05ff06ffff05ff02ffff05ff17ffff01ff80808080808080808080ffff01ffffff05ff17ff2f808080ffff01ffff09808080ff01808080ffff01ffff05ffff05ff04ffff05ff05ffff05ffffff05ff06ffff05ff02ffff05ff17ffff01ff808080808080ffff01ff8080808080ffffff05ff17ff2f8080808080ff01808080ffff05ffff01ffff39ffff05ffff04ffff08ff0580ffff01ffff0bffff01ff0280ffffff05ff06ffff05ff02ffff05ff09ffff01ff808080808080ffffff05ff06ffff05ff02ffff05ff0dffff01ff8080808080808080ffff01ffff0bffff01ff0180ff05808080ff01808080ff0180808080ffff05ffff01ffb08f5c4a3a8ee47f20d3ff05720dd00c2c3433153e8f1ef62a8bb247208b9e12fc17a57e3a8cd6dcd58a720efe0f39747280ff01808080ffff80ffff01ff8080ff80808080ffffa00d33ae69f00749738b8fb2ad99e6cdcd5826660925958ee48533946daa5e4be1ffffffff05ffff01ffffff05ffff01ffffff05ffff04ff0bffff01ffffff05ffff04ffff0aff05ffff12ff0bffff13ffff0bff0bffffff05ff06ffff05ff02ffff05ff17ffff01ff80808080808080808080ffff01ffffff05ff17ff2f808080ffff01ffff09808080ff01808080ffff01ffff05ffff05ff04ffff05ff05ffff05ffffff05ff06ffff05ff02ffff05ff17ffff01ff808080808080ffff01ff8080808080ffffff05ff17ff2f8080808080ff01808080ffff05ffff01ffff39ffff05ffff04ffff08ff0580ffff01ffff0bffff01ff0280ffffff05ff06ffff05ff02ffff05ff09ffff01ff808080808080ffffff05ff06ffff05ff02ffff05ff0dffff01ff8080808080808080ffff01ffff0bffff01ff0180ff05808080ff01808080ff0180808080ffff05ffff01ffb08f5c4a3a8ee47f20d3ff05720dd00c2c3433153e8f1ef62a8bb247208b9e12fc17a57e3a8cd6dcd58a720efe0f39747280ff01808080ffff80ffff01ff
```

Comment:
And this TODO comment could be removed. Although, we should probably stick this program in a file, loaded by the test rather than in a string literal.